[2024-09-24T00:45:15.185+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_ingestion_15min.trigger_stage_processing scheduled__2024-09-24T00:30:00+00:00 [queued]>
[2024-09-24T00:45:15.195+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_ingestion_15min.trigger_stage_processing scheduled__2024-09-24T00:30:00+00:00 [queued]>
[2024-09-24T00:45:15.195+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-09-24T00:45:15.209+0000] {taskinstance.py:1382} INFO - Executing <Task(TriggerDagRunOperator): trigger_stage_processing> on 2024-09-24 00:30:00+00:00
[2024-09-24T00:45:15.214+0000] {standard_task_runner.py:57} INFO - Started process 2695 to run task
[2024-09-24T00:45:15.216+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'raw_ingestion_15min', 'trigger_stage_processing', 'scheduled__2024-09-24T00:30:00+00:00', '--job-id', '566', '--raw', '--subdir', 'DAGS_FOLDER/raw_ingestion_15min.py', '--cfg-path', '/tmp/tmpzc8r1pah']
[2024-09-24T00:45:15.217+0000] {standard_task_runner.py:85} INFO - Job 566: Subtask trigger_stage_processing
[2024-09-24T00:45:15.321+0000] {task_command.py:416} INFO - Running <TaskInstance: raw_ingestion_15min.trigger_stage_processing scheduled__2024-09-24T00:30:00+00:00 [running]> on host airflow-worker
[2024-09-24T00:45:15.499+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='raw_ingestion_15min' AIRFLOW_CTX_TASK_ID='trigger_stage_processing' AIRFLOW_CTX_EXECUTION_DATE='2024-09-24T00:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-24T00:30:00+00:00'
[2024-09-24T00:45:15.605+0000] {trigger_dagrun.py:173} INFO - Clearing stage_processing on 2024-09-24T00:00:00+00:00
[2024-09-24T00:45:15.800+0000] {trigger_dagrun.py:209} INFO - Waiting for stage_processing on 2024-09-24 00:00:00+00:00 to become allowed state ['success'] ...
[2024-09-24T00:45:25.845+0000] {trigger_dagrun.py:209} INFO - Waiting for stage_processing on 2024-09-24 00:00:00+00:00 to become allowed state ['success'] ...
[2024-09-24T00:45:35.880+0000] {trigger_dagrun.py:209} INFO - Waiting for stage_processing on 2024-09-24 00:00:00+00:00 to become allowed state ['success'] ...
[2024-09-24T00:45:45.921+0000] {trigger_dagrun.py:209} INFO - Waiting for stage_processing on 2024-09-24 00:00:00+00:00 to become allowed state ['success'] ...
[2024-09-24T00:45:55.986+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/trigger_dagrun.py", line 220, in execute
    raise AirflowException(f"{self.trigger_dag_id} failed with failed states {state}")
airflow.exceptions.AirflowException: stage_processing failed with failed states failed
[2024-09-24T00:45:55.989+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=raw_ingestion_15min, task_id=trigger_stage_processing, execution_date=20240924T003000, start_date=20240924T004515, end_date=20240924T004555
[2024-09-24T00:45:56.010+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 566 for task trigger_stage_processing (stage_processing failed with failed states failed; 2695)
[2024-09-24T00:45:56.050+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-09-24T00:45:56.094+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
